CMakeLists.txt

```diff
@@ -46,6 +46,10 @@ if(POLICY CMP0177)
     cmake_policy(SET CMP0177 NEW)
 endif()
 
+# set CMAKE_POLICY_VERSION_MINIMUM to avoid errors from 3rd party dependencies
+# after cmake is updated to 4.0
+set(CMAKE_POLICY_VERSION_MINIMUM 3.5)
```

Comment:
@olpipi do you think we can move this variable setting closer to a place where 3rd party dependency is added to OV project? This will minimize impact.